build_scripts/appimage/AppRun

```diff
@@ -12,6 +12,17 @@ if [ "$1" == "service" ] ; then
     exec /bin/bash "${APPDIR}/install-infection-monkey-service.sh" ${@:2}
 fi
 
+
+# Check if running as root
+if [ $(id -u) -eq 0 ]; then
+    echo "Warning: running as root is not recommended. You should instead run the AppImage as a regular user."
+    echo "If you want to run the Agent as root, do so by choosing manual run mode."
+else
```

Comment:
This clause should only run if the executable doesn't already have cap_net_bind_service. As written, this will prompt the user every time the Island is started.

Reply:
Prompt for what? Maybe we can remove the `sudo` from `getcap`